ui/app/controllers/servers/server.js

```diff
@@ -1,10 +1,20 @@
 import Ember from 'ember';
 
-const { Controller } = Ember;
+const { Controller, computed } = Ember;
 
 export default Controller.extend({
   activeTab: 'tags',
 
+  sortedTags: computed('model.tags', function() {
```

Comment:
will you need to update this later with `@each` once you have polling happening?

Reply:
Unfortunately `@each` doesn't apply here since it's an object, but that was a concern I had. I think it will be fine, since a poll should update the entire object reference. If not, then I will have to transform the object into an array in the serializer.